test/new-e2e/tests/agent-runtimes/checks/network/common.go

```diff
@@ -0,0 +1,101 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// Package checknetwork contains tests for the network check
+package checknetwork
+
+import (
+	"math"
+
+	gocmp "github.com/google/go-cmp/cmp"
+	gocmpopts "github.com/google/go-cmp/cmp/cmpopts"
+	"github.com/stretchr/testify/require"
+
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/e2e"
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/environments"
+	awshost "github.com/DataDog/datadog-agent/test/new-e2e/pkg/provisioners/aws/host"
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/testcommon/check"
+	checkUtils "github.com/DataDog/datadog-agent/test/new-e2e/tests/agent-runtimes/checks/common"
+	e2eos "github.com/DataDog/test-infra-definitions/components/os"
+	"github.com/DataDog/test-infra-definitions/scenarios/aws/ec2"
+)
+
+type networkCheckSuite struct {
+	e2e.BaseSuite[environments.Host]
+	descriptor            e2eos.Descriptor
+	metricCompareFraction float64
+	metricCompareDecimals int
+}
+
+func (v *networkCheckSuite) getSuiteOptions() []e2e.SuiteOption {
+	suiteOptions := []e2e.SuiteOption{}
+	suiteOptions = append(suiteOptions, e2e.WithProvisioner(
+		awshost.Provisioner(
+			awshost.WithEC2InstanceOptions(ec2.WithOS(v.descriptor)),
+		),
+	))
+
+	return suiteOptions
+}
+
+func (v *networkCheckSuite) TestNetworkCheck() {
+	testCases := []struct {
+		name        string
+		checkConfig string
+		agentConfig string
+	}{
+		{
+			"default",
+			`init_config:
+instances:
+  - use_mount: false
+`,
+			``,
+		},
+	}
```

Comment:
Ideally we should add more configurations to test (same for the disk check btw) but not a blocker